.env.development

```diff
@@ -2,15 +2,15 @@ NODE_ENV=development
 PORT=3000
 BASE_REDIRECT_URL=http://localhost:3000/u/
 
-########## InMemory Storage ##########
+########## Storage ##########
+#### InMemory
```

Comment:
pretty obvious its InMemory from the value. I think this comment can be removed.

maybe put in comment something like ###InMemory \ Redis  \ Mongo  etc...

Reply:
Fixed